benchmark/src/main/java/com/distkv/drpc/benchmark/common/consumer/DubboHttpProcessHandler.java

```diff
@@ -0,0 +1,90 @@
+package com.distkv.drpc.benchmark.common.consumer;
```

Comment:
What's the purpose to use dubbo http process?

Does it mean we use dubbo rpc via HTTP?